pkg/northbound/gnmi/v2/service.go

```diff
@@ -79,15 +79,17 @@ type Server struct {
 
 // Capabilities implements gNMI Capabilities
 func (s *Server) Capabilities(ctx context.Context, req *gnmi.CapabilityRequest) (*gnmi.CapabilityResponse, error) {
-	// TODO adds capability
-	/*capabilities, err := s.modelRegistry.Capabilities()
-	if err != nil {
-		return nil, err
-	}*/
+	plugins := s.pluginRegistry.GetPlugins()
+
+	supportedModels := make([]*gnmi.ModelData, 0)
+	for _, plugin := range plugins {
+		capabilities := plugin.Capabilities(ctx, nil)
```

Comment:
What is the second argument here which can be nil?

Reply:
Not sure what the intention is - in the function itself the parameter is unused

Reply:
OK perhaps we should look into what it was before and remove or complete it if it is needed. Going to approve it that you can merge.

Reply:
OK. I'll find out of we can remove that parameter before I merge it.